api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb

```diff
@@ -36,5 +36,86 @@ def count
       organizations_count = MnoEnterprise::Tenant.get('tenant').organizations_count
       render json: {count: organizations_count }
     end
+
+    # POST /mnoe/jpi/v1/admin/organizations
+    def create
+      # Create new organization
+      @organization = MnoEnterprise::Organization.create(organization_update_params)
+
+      # OPTIMIZE: move this into a delayed job?
+      update_app_list
+
+      @organization_active_apps = @organization.app_instances
+
+      render 'show'
+    end
+
+    # POST /mnoe/jpi/v1/admin/organizations/1/users
+    # Invite a user to the organization (and create it if needed)
+    # This does not send any emails (emails are manually triggered later)
+    def invite_member
+      @organization = MnoEnterprise::Organization.find(params[:id])
+
+      # Find or create a new user - We create it in the frontend as MnoHub will send confirmation instructions for newly
+      # created users
+      user = MnoEnterprise::User.find_by(email: user_params[:email]) || create_unconfirmed_user(user_params)
+
+      # Create the invitation
+      invite = @organization.org_invites.create(
+          user_email: user.email,
+          user_role: params[:user][:role],
+          referrer_id: current_user.id,
+          status: 'staged' # Will be updated to 'accepted' for unconfirmed users
+      )
+
+      @user = user.confirmed? ? invite : user.reload
+    end
+
+    protected
+      def organization_permitted_update_params
+        [:name]
+      end
+
+      def organization_update_params
+        params.fetch(:organization, {}).permit(*organization_permitted_update_params)
+      end
+
+      def user_params
+        params.require(:user).permit(:email, :name, :surname, :phone)
+      end
+
+      # Create an unconfirmed user and skip the confirmation notification
+      # TODO: monkey patch User#confirmation_required? to simplify this? Use refinements?
+      def create_unconfirmed_user(user_params)
+        user = MnoEnterprise::User.new(user_params)
+        user.skip_confirmation_notification!
+        user.save
+
+        # Reset the confirmation field so we can track when the invite is send - #confirmation_sent_at is when the confirmation_token was generated (not sent)
+        # Not ideal as we do 2 saves, and the previous save trigger a call to the backend to validate the token uniqueness
+        user.assign_attributes(confirmation_sent_at: nil, confirmation_token: nil)
+        user.save
+        user
+      end
```

Comment:
From the devise confirmable model:
```ruby
before_create :generate_confirmation_token, if: :confirmation_required?
after_create  :send_on_create_confirmation_instructions, if: :send_confirmation_notification?
```

`.skip_confirmation_notification!` only skip the 2nd callback and not the first one so I have to reset the fields.

I could monkey patch devise `#generate_confirmation_token` to exit if some flag is set if you think that's better.